test/checkArrayExpectation.js

```diff
@@ -9,7 +9,7 @@ module.exports = function checkArrayExpectation(testDirectory, object, kind, fil
 	}
 	var array = object[kind + "s"].slice().sort();
 	if(kind === "warning") array = array.filter(function(item) {
-		return !/from UglifyJs/.test(item);
+		return !/from UglifyJs|was applied \d+ times/.test(item);
```

Comment:
Check out this PR which will.contain a readme for how to add tests. In this case you'd add a test to configCases